lib/tasks/pubmed.rake

```diff
@@ -3,6 +3,76 @@ namespace :pubmed do
     @client ||= PubmedClient.new
   end
 
+  desc 'update pubmed source records and rebuild pub_hashes for publications for a given list of cap_profile_ids passed in a text file'
+  # filename should be a list of cap_profile_id, one per line with no header
+  # call with RAILS_ENV=production bundle exec rake pubmed:refresh_source_records_for_cap_profile_ids['filename.txt']
+  task :refresh_source_records_for_cap_profile_ids, [:filename] => :environment do |_t, args|
+    filename = args[:filename]
+    fail "filename is required." unless filename.present?
+    cap_profile_ids=File.readlines(filename)
```

Comment:
Do you mean validate that the cap_author_ids are valid?  That is checked later (invalid cap author ids are just skipped in the loop rather than validating all upfront)